alfresco-health-processor-platform/src/main/amp/config/alfresco/subsystems/HealthProcessor/default/healthprocessor.properties

```
@@ -21,13 +21,12 @@ eu.xenit.alfresco.healthprocessor.plugin.content-validation.enabled=false
 eu.xenit.alfresco.healthprocessor.plugin.content-validation.properties=
 
 eu.xenit.alfresco.healthprocessor.plugin.solr-index.enabled=false
+eu.xenit.alfresco.healthprocessor.plugins.solr-index.check-transaction=false
```

Ah, I missed previously that your property was named "plugins" instead of "plugin".

```suggestion
eu.xenit.alfresco.healthprocessor.plugin.solr-index.check-transaction=false
```